assets/css/smallscreen.css

```diff
@@ -0,0 +1,92 @@
+@media screen and (max-width: 600px) {
+  body {
+    margin: 0;
+    background-color: rgb(255, 255, 255);
+  }
+
+  html {
+    font-family: "Lucida Sans", sans-serif;
+  }
+
+  .header {
+    padding: 17px 15px 15px 15px;
+    height: 72px;
+    background-color: #3c3a39;
+    color: #acb7c3;
+    position: relative;
+    max-width: 100vw;
+  }
+
+  .header nav {
+    top: 10px;
+    flex-direction: row;
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .header nav .logo {
+    font-size: 20px;
+  }
```

Comment:
- [x] Good job working on the styles but still the header is a little different from the requirements. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Please fix the size of your header elements heading and hamburger icon and make sure it matches the design.

Your design

![Screenshot (55)](https://user-images.githubusercontent.com/83803414/153378297-c93c9483-8db4-43f4-93dd-67f6fef1b181.png)


Figma design

![Screenshot (56)](https://user-images.githubusercontent.com/83803414/153378265-8734f7b3-dec3-4405-bede-e988f54c6791.png)